extensions/CocoStudio/Reader/SceneReader.h

```diff
@@ -62,8 +62,11 @@ class CC_EX_DLL SceneReader
 {
 
 public:
-	static SceneReader* sharedSceneReader();
-	static void purge();
+	CC_DEPRECATED_ATTRIBUTE static SceneReader* sharedSceneReader();
+	static SceneReader* getInstance();
+	CC_DEPRECATED_ATTRIBUTE static void purge();
+	static void destroyInstance();
+	static void destroyCocosStudio();
```

Comment:
It is a strange name.
SceneReader::destroyCocosStudio().
Why not name destroyInstance()?